flags.go

```diff
@@ -20,24 +20,34 @@ var supportedTLSVersions map[string]uint16 = map[string]uint16{
 	"1.3": tls.VersionTLS13,
 }
 
-type tlsMinVersionValue struct {
-	version     *string
-	versionCode *uint16
+type tlsVersionValue struct {
+	version uint16
 }
```

Comment:
Could it be directly an int?
```suggestion
type tlsVersionValue uint16
```